realm-jni/src/io_realm_internal_SharedGroup.cpp

```diff
@@ -53,27 +77,13 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_SharedGroup_nativeCreate(
         }
         else {
             SharedGroup::DurabilityLevel level;
-            if (durability == 0)
-                level = SharedGroup::durability_Full;
-            else if (durability == 1)
-                level = SharedGroup::durability_MemOnly;
-            else if (durability == 2)
-#ifdef _WIN32
-                level = SharedGroup::durability_Full;   // For Windows, use Full instead of Async
-#else
-                level = SharedGroup::durability_Async;
-#endif
-            else {
+            if (!jintToDurabilityLevel(durability, level)) {
                 ThrowException(env, UnsupportedOperation, "Unsupported durability.");
```

Comment:
I convinced myself to throw the exception in the local function since i found in the other PR i have to send a long description exception two times...
And also, the name changed to jint_to_durability_level since it seems to be the right naming style.